build.ps1

```diff
@@ -68,6 +68,20 @@ Invoke-BuildStep 'Restoring solution packages' { `
     } `
     -skip:$SkipRestore `
     -ev +BuildErrors
+    
+Invoke-BuildStep 'Set version metadata in AssemblyInfo.cs' {
+        $assemblyInfos = `
+            "src\NuGet.Indexing\Properties\AssemblyInfo.g.cs", `
```

Comment:
Should this use PSScriptRoot like Gallery's build, in case it's not run from the script directory?